config/routes.rb

```diff
@@ -3,5 +3,5 @@
   get "signin" => "sessions#new", as: :signin
   post "signin" => "sessions#create"
   get "signout" => "sessions#destroy", as: :signout
-  resources :registrations, only: [:new, :create]
+  resources :registrations
 end
```

Comment:
@ipbrennan90 I think you missed the boat on this one a little. The point was to use a combination of RSpec and Capybara to write a test that validates the defect. You solved it correctly, but missed the test part.